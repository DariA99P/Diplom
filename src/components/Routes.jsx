import React from 'react';

import Loadable from 'react-loadable';
import { Switch, Route } from 'react-router-dom';
import Loader from 'components/shared/loader';

const Welcome = Loadable({
  loader: () => import('components/pages/welcome'),
  loading: Loader,
});

const NotFound = Loadable({
  loader: () => import('components/pages/notFound'),
  loading: Loader,
});

const Routes = () => (
  <Switch>
    <Route exact path="/" component={Welcome} />
    <Route component={NotFound} />
  </Switch>
);

export default Routes;
