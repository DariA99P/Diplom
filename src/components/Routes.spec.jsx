import React from 'react';
import { shallow } from 'enzyme';

import Routes from './Routes';

describe('<Routes />', () => {
  it('should render properly', () => {
    const tree = shallow(<Routes />);

    expect(tree).toMatchSnapshot();
  });
});
