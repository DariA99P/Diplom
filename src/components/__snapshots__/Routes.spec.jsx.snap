// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Routes /> should render properly 1`] = `
<Switch>
  <Route
    component={[Function]}
    exact={true}
    path="/"
  />
  <Route
    component={[Function]}
  />
</Switch>
`;
